pkg/build/node/const.go

```diff
@@ -27,9 +27,10 @@ const (
 The default CNI manifest and images are our own tiny kindnet
 */
 
-var defaultCNIImages = []string{"kindest/kindnetd:0.1.0"}
+var defaultCNIImages = []string{"kindest/kindnetd:0.1.0", "gcr.io/bentheelder-kind-dev/ip-masq-agent:v2.3.0-1-g51408b3"}
```

Comment:
https://github.com/kubernetes-incubator/ip-masq-agent/blob/e54f921c3191d303a708d6a9dbf88fc8d14d6c26/cmd/ip-masq-agent/ip-masq-agent.go#L224 , it doesn´t accept IPv6 prefixes

🤔 do you think that the project will accept patches to make it IPv6 compatible?

Reply:
if they're low risk I'd assume so, otherwise we can do some fixes ourselves.

Reply:
spoke to @MrHohn who has been very gracious accepting PRs so far and getting images pushed, we should be good to send fixes for that. I'm sending one right now for that particular issue.

Reply:
https://github.com/kubernetes-incubator/ip-masq-agent/pull/39

Reply:
updated to use the upstream image